### DISCUSSION TITLE
Swell multi-vendor store development

### DISCUSSION THREAD

**Discussion by gfdir** [swellstores/community#404] [Help]
Hi all,
Please pardon any faux pas on my part. I'm new to Swell and Github. I found this community through Swell as I searched for a solution to develop a multi-vendor platform. I noted a Swell does in fact have one built: [spinn coffee roasters](https://www.spinn.com/). So I'm here to ask how I can find a Dev to build a multi-vendor marketplace platform for my project. 

I know people who are dev's, but I'd rather work with someone already familiar with Swell and has done previous development. 

Thank a ton in advance.

**Comment by ericingram** [comment]
Calling all devs!
